eth-contracts/contracts/staking/Staking.sol

```diff
@@ -7,12 +7,12 @@ import "@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/SafeERC20.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20Burnable.sol";
-import "../res/IsContract.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/utils/Address.sol";
```

Comment:
I guess there's a bytecode consideration to importing Address contract vs just vendoring a tiny IsContract one, but then we have to get that audited etc, plus does't look like we have any bytecode restrictions now